tests/main.test.js

```diff
@@ -63,3 +63,9 @@ test('handles 404 CSS file', async () => {
     expect(e.message).toMatch('404 on')
   }
 })
+
+test('media queries print removed', async () => {
```

Comment:
I have test (in big PR) that shows that print media queries stays.

Reply:
I know and I think that's "wrong". I.e. the `@media print ...` blocks *should* be removed. 
I suggest you remove "your version" and then merging with this PR will be easier for you.

Reply:
I mean my tests are stating facts about current code - exposing right and wrong cases, there are more wrong cases. As soon as you fix code you update tests. I just referred to the fact that there will be merge conflict nothing else